### PR TITLE
Make location lookup look at existing users' coordinates as a last resort

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**
  
Make location lookup look at existing users' coordinates as a last resort. Also apply retroactively!

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by michielderoos** [comment]
[GPS location only seems to be working on one user uploaded with that location](https://trello.com/c/8TsmB0ii/691-gps-location-only-seems-to-be-working-on-one-user-uploaded-with-that-location)

**Review by tristanhcole — APPROVED** [review]
Makes sense to me! 👍

**Comment by tristanhcole** [comment]
HAHAHA. I tested this with "banana" as the location set to Melbourne but apparently, there is a town called banana in Australia. https://www.google.com/maps/place/Banana+QLD+4702/data=!4m2!3m1!1s0x6bc13d589f58a60f:0x400eef17f20ccc0?sa=X&ved=2ahUKEwiAlNXZs73tAhXH4zgGHQKRC4gQ8gEwG3oECCcQAQ

File: app/migrations/seed.py [hunk 1]
```diff
@@ -402,4 +421,6 @@ def create_float_transfer_account(app):
 
     create_float_transfer_account(current_app)
 
+    refresh_user_locations(current_app)
+
```
[the remark this answers is not in the document]
Comment (michielderoos) [reply app/migrations/seed.py:425]: I think so. It's not a task well suited to migrations, and we have a [task](https://trello.com/c/UPUuNrvO/708-remove-lat-lng-backfill-from-seedpy) queued to remove it once this PR makes it into prod

**Pull request merged**